virtool/hmm/api.py

```diff
@@ -191,6 +192,9 @@ async def get_hmm_annotations(req):
 
     Fetches a compressed json file containing the database documents for all HMMs.
     """
+    hmm_path = req.app["config"].data_path / "hmm"
```

Comment:
Please use `get_config_from_app` here.

It maintains typing automatically since the function has a return type annotation. This is not possible with dictionary style access like `["config"]`.
```